MetaMask/metamask-extension
# feat: enable preview token

## **Description**

This PR introduces the ability for the user to utilize a preview token generated by Contentful to view unpublished content. The PR is particularly useful for the marketing team, allowing them to test product announcements before they are published.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27809?quickstart=1)

## **Related issues**

N/A

## **Manual testing steps**

0. Create a draft feature announcement through contentful.
1. retrieve the preview token from 1password or from marketing team/ @MetaMask/notifications team
2. view the page `chrome-extension://<YOUR-EXTENSION-URL>/home.html?previewToken=000`
3. navigate to the notifications page (you may need to enable notifications).
4. You should be able to see preview/draft feature announcements.

## **Screenshots/Recordings**

N/A

### **Before**

N/A

### **After**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
